rmgpy/reduction/reduction.py

```diff
@@ -268,13 +268,20 @@ def searchTargetIndex(targetLabel, reactionModel):
     """
     Searches for the Species object in the core species
     of the reaction that has the same label as the parameter string.
-    """
 
+    This method does not differentiate between multiple species
```

Comment:
Good catch! It would be better to use unittest to show the issue, doc-string might confuse users.